src/components/appRouter.js

```diff
@@ -343,7 +342,6 @@ define(['loading', 'globalize', 'events', 'viewManager', 'layoutManager', 'skinM
                 hashbang: options.hashbang !== false,
                 enableHistory: enableHistory()
             });
-        }).finally(function () {
```

Comment:
This may end up in the loading indicator not being hidden if an error occurs. We may want to do something like `.catch().then(function() { ... })` to emulate the behavior of `.finally()`.

Reply:
Wasn't entirely sure of how to handle the lack of `.finally()`